packages/control-group/src/index.tsx

```diff
@@ -0,0 +1,251 @@
+    />
+  </svg>
+)
+
+const Control: React.FC<ControlProps> = ({
+  item,
+  textAlign,
+  disabled,
+  invalid,
+  height,
+  value: groupValue,
+  groupAppearance,
+  controlAppearance,
+  isOpen,
+  onChange
+}) => {
+  const isOutline = controlAppearance === 'outline'
+
+  const checked = Array.isArray(groupValue)
+    ? groupValue.includes(item.value || '')
+    : groupValue === item.value
+
+  const renderProps = {
+    textAlign,
+    checked,
+    disabled,
+    invalid,
+    height,
+    ...item
+  }
+  const handleOnChange = (itemValue: string) => {
+    if (Array.isArray(groupValue)) {
+      if (groupValue.includes(itemValue)) {
+        safeInvoke(onChange, groupValue.filter(item => item !== itemValue))
+      } else {
+        safeInvoke(onChange, groupValue.concat(itemValue))
+      }
+    } else {
+      const newValue = groupValue === itemValue ? undefined : itemValue
+      safeInvoke(onChange, newValue)
+    }
+  }
+
+  switch (groupAppearance) {
+    case 'radio-horizontal':
+    case 'radio-vertical':
+      return (
+        <Radio
+          {...renderProps}
+          appearance={controlAppearance as RadioAppearanceType}
+          onChange={() => safeInvoke(handleOnChange, item.value)}
+        >
+          {item.label}
+        </Radio>
+      )
+    case 'checkbox-horizontal':
+    case 'checkbox-vertical':
+      return (
+        <Checkbox
+          {...renderProps}
+          appearance={controlAppearance as CheckboxAppearanceType}
+          onChange={() => safeInvoke(handleOnChange, item.value)}
+        >
+          {item.label}
+        </Checkbox>
+      )
+    case 'button':
+    default:
+      return (
+        <ControlButton
+          {...renderProps}
+          appearance={controlAppearance as ButtonAppearanceType}
+          onClick={() => safeInvoke(handleOnChange, item.value)}
+          activeGroup={Boolean(
+            Array.isArray(groupValue) ? groupValue.length : groupValue
+          )}
+          isOpen={isOpen}
+        >
+          {isOutline ? (
+            <Strong>{item.label}</Strong>
+          ) : (
+            <span>{item.label}</span>
+          )}
+        </ControlButton>
+      )
+  }
+}
+
+const ControlGroupFC: React.FC<ControlGroupProps> = props => {
+  const {height, controlAppearance, groupAppearance} = useDefaults(
+    'controlGroup',
+    props,
+    {
+      height: 32,
+      controlAppearance: 'default' as ControlAppearanceType,
+      groupAppearance: 'button' as ControlGroupAppearanceType
+    }
+  )
+  const {
+    items,
+    onChange,
+    value,
+    layout,
+    textAlign,
+    disabled,
+    invalid,
+    visibleCount
+  } = props
+
+  const [controlsNumber, setControlsNumber] = React.useState(0)
+  const [isOpen, setIsOpen] = React.useState(false)
+  const hasMoreButton =
+    controlAppearance === 'outline' && groupAppearance === 'button'
```

Comment:
This should be supported for all appearances of button, not only outline but it can be done in another PR